adafruit_mpu6050.py

```diff
@@ -347,3 +372,15 @@ def cycle_rate(self, value: int) -> None:
             raise ValueError("cycle_rate must be a Rate")
         self._cycle_rate = value
         sleep(0.01)
+
+    @property
+    def clock_source(self) -> int:
+        """Returns current CLKSEL value, Reg Map Pg 40"""
```

Comment:
I think it might be more helpful, since you reference this in the description of `Clksel`, to describe the actual setting being changed here.  This is also important because this docstring is what will show in the aforementioned ReadTheDocs documentation.

Reply:
It should be about this short though.